src/index.js

```diff
@@ -33,15 +34,17 @@ export class Inspector {
     }
     dispatch(_node, "update");
   }
-  rejected(error) {
+  rejected(error, name) {
     const {_node} = this;
     _node.classList.remove("observablehq--running");
     _node.classList.add("observablehq--error");
     while (_node.lastChild) _node.removeChild(_node.lastChild);
-    var span = document.createElement("span");
-    span.className = "observablehq--inspect";
+    var div = document.createElement("div");
+    div.className = "observablehq--inspect";
+    if (name) div.appendChild(inspectName(name));
+    var span = div.appendChild(document.createElement("span"));
```

Comment:
I think you can get away with just a text node here and drop the span if desired?